packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -29,28 +33,62 @@ export const styles = (theme) => ({
     ...theme.typography.body2,
     minHeight: 'auto',
   },
+  indicatorWrapper: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  indicator: {
+    marginLeft: theme.spacing(2),
+  },
 });
 
 const MenuItem = React.forwardRef(function MenuItem(props, ref) {
   const {
+    children: childrenProp,
     classes,
     className,
     component = 'li',
     disableGutters = false,
     ListItemClasses,
+    openSubMenu = false,
+    onKeyDown,
     role = 'menuitem',
     selected,
+    subMenu,
+    SubMenuIcon = KeyboardArrowRight,
```

Comment:
I think this should be `subMenuIcon`. Look at how TreeView defined its icons": https://material-ui.com/api/tree-view/

Reply:
Sounds good to me. 
```suggestion
    subMenuIcon = KeyboardArrowRight,
```